### PR TITLE
Update zeus.ms

### PR DESCRIPTION
**Pull request by flamingwarrior10**
zeus remake

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/buffs/zeus.ms [hunk 1]
```diff
@@ -9,6 +9,24 @@
             'trail': '',
             'op': 'true',
             'speech': "Zeus Almighty.",
-            'powers': array(),
+            'powers': array(
+                "/buff",
+                "/lightning",
+                "/gmc",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/zeus.ms]: need evidence of matter manipulation

File: CommandHelper/LocalPackages/commands/buffs/zeus.ms [hunk 2]
```diff
@@ -9,6 +9,24 @@
...
+                "/fly",
+                "/kill,"
+                "/disguise",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/zeus.ms]: need evidence of disguises

Comment (flamingwarrior10) [inline CommandHelper/LocalPackages/commands/buffs/zeus.ms]: https://en.wikipedia.org/wiki/Shapeshifting

File: CommandHelper/LocalPackages/commands/buffs/zeus.ms [hunk 3]
```diff
@@ -9,6 +9,24 @@
             'trail': '',
             'op': 'true',
             'speech': "Zeus Almighty.",
-            'powers': array(),
+            'powers': array(
+                "/buff",
+                "/lightning",
+                "/gmc",
+                "/fly",
+                "/kill,"
```
Comment (flamingwarrior10) [inline CommandHelper/LocalPackages/commands/buffs/zeus.ms]: Scales of Zeus Weighs soul to determine who dies or not in actuality Zeus controls the scales

**Pull request merged**